app/views/audits/audits/_free_text_question.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="row">
+<div class="row <% unless show_hide %>if-js-hide<% end %> t-<%= attribute.to_s.dasherize %> free-text-question">
```

Comment:
`show_hide` is an oxymoron - could you please clarify what you're actually showing?